thci/OTNCS.py

```diff
@@ -3602,9 +3607,6 @@ def _cliReadLine(self):
             return self.__lines.pop(0)
 
     def _cliWriteLine(self, line):
-        # NCS adaptation
```

Comment:
I think this was added for a reason. Might be breaking some Thread 1.1 certification test by removing those lines.

Reply:
restored

Reply:
Doesn't this interfere with the BBR operation?

Reply:
OTNCS class is not used by OTNCS_BR class. The second has it's own implementation.